### DISCUSSION TITLE
Does the new method use regularization images?

### DISCUSSION THREAD

**Discussion by Gyro0o** [TheLastBen/fast-stable-diffusion#540] [Q&A]
Does the new method use regularization images
and how could I input my own dataset/change the current one?

**Comment by TheLastBen** [comment]
There is the option "Contains_faces", set it to Male, Female or a Mix.

You should know that regularization images doesn't help much with the quality of the results, they actually require more training steps to get decent results, I recommend training without reg images ("Contains_faces" set to "No")

**Reply by jysung** [reply]
But does "contian faces" help eventually when trained with more training steps? If not, when is that useful for?

**Reply by TheLastBen** [reply]
it's useful when training a lot of subjects, it helps reduce the leaking from a subject to another, other than that, it is preferable to set it to No

**Comment by Gyro0o** [comment]
I see
I still hope u add an option under "Contains_faces" to add our own directory like the old method 
Cuz I had a good dataset for styles and I was getting good results with it in both joepenna and your old method

**Reply by TheLastBen** [reply]
old method : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Dreambooth/fast_DreamBooth-Old-Method.ipynb

**Reply by Gyro0o** [reply]
Can I train multiple subjects in the old method as good as the new one?
That would be the perfect dreambooth

**Reply by TheLastBen** [reply]
yes, check the box image captioning, and caption the images

**Reply by Gyro0o** [reply]
Cool
In the old method there was no text_encoder_training box
Can I uncheck that?

**Reply by TheLastBen** [reply]
if you caption the images, uncheck the text_encoder
